pinot-query-planner/src/main/java/org/apache/pinot/query/planner/plannode/ExchangeNode.java

```diff
@@ -46,11 +47,14 @@ public class ExchangeNode extends AbstractPlanNode {
   @ProtoProperties
   private List<RelFieldCollation> _collations;
 
+  @ProtoProperties
+  private Set<String> _tableNames;
+
   public ExchangeNode(int planFragmentId) {
     super(planFragmentId);
   }
 
-  public ExchangeNode(int currentStageId, DataSchema dataSchema, RelDistribution distribution,
+  public ExchangeNode(int currentStageId, DataSchema dataSchema, Set<String> tableNames, RelDistribution distribution,
```

Comment:
what is tableNames doing here? and can we add some comment on what this means?

Reply:
It carries the tables to scan under this ExchangeNode.